db/version_edit_handler.cc

```diff
@@ -638,6 +638,8 @@ void VersionEditHandlerPointInTime::CheckIterationResult(
         versions_.erase(v_iter);
       }
     }
+  } else {
+    versions_.clear();
```

Comment:
Do we also have to `delete` the raw `Version` pointers stored in the map here?

Reply:
Done